Tests/js/models/assets/ez-trashitemmodel-tests.js

```diff
@@ -257,6 +257,38 @@ YUI.add('ez-trashitemmodel-tests', function (Y) {
 
             Mock.verify(this.contentServiceMock);
         },
+
+        "Should call the api on restore with destination provided": function () {
+            var restoreCallback = function () {},
+                destination = {};
+
+            this.options = {api: this.apiMock, destination: destination};
+
+            Mock.expect(this.contentServiceMock, {
+                method: "recover",
+                args: [this.id, destination, Mock.Value.Function],
+                run: Y.bind(function (id, options, callback) {
+                    Assert.areSame(
```

Comment:
actually, the mistake here is that `options` should be called `destination` and it is already asserted automatically in the `args`.